### ISSUE TITLE
Corgis are not created and displayed like expected

### ISSUE THREAD

**Issue by AnaisUrlichs** [nearprotocol/corgis#7]
- [x] When creating a new corgi, the screen is stuck on the animation. Going back to the main page the corgi has been created;

Expected behaviour: After creating a corgi, I am led to a separate screen that displays my corgi with an option to create a new corgi or view all corgis on the main page.

- [x] After creating a new corgi, the corgi-count in the top is not updated;

Expected behaviour: After creating a new corgi, I see how many corgis I have in the top count

**Comment by mikedotexe** [comment]
@AnaisUrlichs I believe I saw the same behavior before, but it was only appearing when running corgis locally. Did you also observe this behavior when running it locally?

**Comment by AnaisUrlichs** [comment]
@mikedotexe yes part of it -- the loading error after generating a corgi is only locally but in either case, the collection of user-owned corgis is not displayed after generating a new corgi
